Stripe open throws exception

Stripe open throws exception.

<img width="1265" alt="Screen Shot 2019-08-28 at 1 53 48 PM" src="https://user-images.githubusercontent.com/53238203/63892279-17081e80-c99c-11e9-9779-57fc205e8d0f.png">

## Steps:
1. Install CLI
1. Run `stripe open`

## Actual
```
stripe open
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/stripe/stripe-cli/pkg/cmd.(*openCmd).runOpenCmd(0xc0000940d0, 0xc0001b2780, 0x19e9fd8, 0x0, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/stripe/stripe-cli/pkg/cmd/open.go:121 +0x4be
github.com/spf13/cobra.(*Command).execute(0xc0001b2780, 0x19e9fd8, 0x0, 0x0, 0xc0001b2780, 0x19e9fd8)
        /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0x19c4da0, 0x6bd, 0xc0000c5f68, 0x147e2f8)
        /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fc
github.com/spf13/cobra.(*Command).Execute(...)
        /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/stripe/stripe-cli/pkg/cmd.Execute()
        /home/travis/gopath/src/github.com/stripe/stripe-cli/pkg/cmd/root.go:59 +0x9e
main.main()
        /home/travis/gopath/src/github.com/stripe/stripe-cli/cmd/stripe/main.go:6 +0x20
```

## Expected
No error and list of available open commands.

--- a/pkg/cmd/open.go
+++ b/pkg/cmd/open.go
@@ -108,7 +108,8 @@ func (oc *openCmd) runOpenCmd(cmd *cobra.Command, args []string) error {
 	if list || len(args) == 0 {
 
 		fmt.Println("open quickly opens Stripe pages. To use, run 'stripe open <shortcut>'.")
-		fmt.Println("open supports the following shortcuts:\n")
+		fmt.Println("open supports the following shortcuts:")
+		fmt.Println()
 		shortcuts := openNames()
 		sort.Strings(shortcuts)
 		longest := getLongestShortcut(shortcuts)

--- a/pkg/cmd/open.go
+++ b/pkg/cmd/open.go
@@ -105,11 +105,16 @@ func (oc *openCmd) runOpenCmd(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if list {
+	if list || len(args) == 0 {
+
+		fmt.Println("open quickly opens Stripe pages. To use, run 'stripe open <shortcut>'.")
+		fmt.Println("open supports the following shortcuts:\n")
 		shortcuts := openNames()
 		sort.Strings(shortcuts)
 		longest := getLongestShortcut(shortcuts)
 
+		fmt.Println(fmt.Sprintf("%s%s", padName("shortcut", longest), "    url"))
+		fmt.Println(fmt.Sprintf("%s%s", padName("--------", longest), "    ---------"))
 		for _, shortcut := range shortcuts {
 			paddedName := padName(shortcut, longest)
 			fmt.Println(fmt.Sprintf("%s => %s", paddedName, nameURLmap[shortcut]))

